### PR TITLE
Add tooling guides tip about JDK 11 native image generation from Docker

### PR DESCRIPTION
**Pull request by gwenneg**
Partially addresses #7014.

I'm not sure it's ok to mention the full `ubi-quarkus-native-image` URL like that in the doc, but this is only a temporary tip since the JDK 8 support will be dropped in Quarkus in the near future hopefully.

@gsmet WDYT?

### CODE REVIEW

**Comment by gwenneg** [comment]
FYI, I checked that the `{graalvm-version}` string is correctly replaced by the GraalVM version when the doc is built:

image

I also noticed the Gradle tooling guide contains deprecated build instructions using `QuarkusNative` which should have been deprecated at the same time as `NativeImageMojo`. I posted [something about it on Zulip](https://quarkusio.zulipchat.com/#narrow/stream/187030-users/topic/configure.20builderImage.20via.20gradle/near/187596485), waiting for a confirmation now.

**Review by gastaldi — APPROVED** [review]

**Comment by geoand** [comment]
We should certainly update the Gradle tooling docs as well to no longer point to the Gradle tasks

**Comment by gwenneg** [comment]
@geoand I agree, but it looks like there's more to do than simply update the doc. For example, if `QuarkusNative` is officially deprecated, we have to do something about `QuarkusTestNative` which depends on `QuarkusNative`. I'd rather have this subject treated in another PR :)

**Comment by gwenneg** [comment]
I created #7104 for that purpose.

Let's merge this PR if that's ok with everyone.

**Pull request merged**
